content/blog/how-the-search-works/index.md

```diff
@@ -0,0 +1,648 @@
++++
+title = "Search on a static site"
+description = "Light weight search results with the Zola static site framework"
+date = 2020-11-16
+draft = true
+
+[taxonomies]
+technologies = ["mithril.js", "Zola", "elasticlunr", "javascript"]
+tags = ["search"]
++++
+
+[Zola](https://www.getzola.org/) is an awesome static site generator written in
+Rust. With Zola you can render a statically generated search index file, but
```

Comment:
This sentence needs more context about why only having the search index is bad and what we will have to build to make it usable.